src/js/client.ts

```diff
@@ -41,34 +38,16 @@ export class ReactNativeClient extends BaseClient<ReactNativeClientOptions> {
    * @param options Configuration options for this SDK.
    */
   public constructor(options: ReactNativeClientOptions) {
-    if (!options.transport) {
```

Comment:
`option.transport` is required and has a fallback in `sdk.tsx`, so this if statement wasn't working